test-src/manageamiset_test.js

```diff
@@ -85,6 +87,11 @@ describe('AMI Set api', () => {
     });
   });
 
+  it('should be able to check if the AMIs from an AMI set are valid', async () => {
+    await client.createAmiSet(id, amiSetDefinition);
+    assume(await client.validateAmiSet(id)).to.deeply.equal([]);
```

Comment:
I wanted to test a known invalid amiSet here, but I didn't knkow how to do it since with the validation method we cannot create "fake" amiSets anymore.
I tried to do something like `Amiset.validate(context, fakeAmiSet)`, but I don't know what to pass as `context`.